website/tests/api/sequences.spec.ts

```diff
@@ -5,7 +5,7 @@ import { getTestSequences } from '../util/testSequenceProvider.ts';
 test.describe('The sequences endpoint', () => {
     const testSequenceEntry = getTestSequences().testSequenceEntry;
 
-    test('should have usable logout button', async ({ request }) => {
```

Comment:
Woops that was a copy/paste leftover I suppose 😄

Reply:
Yes, this confused me for a bit, making me wonder how this update could possibly break the logout button